client/main.lua

```diff
@@ -1,14 +1,14 @@
-local QBCore = exports['qbx-core']:GetCoreObject()
-local PlayerData = QBCore.Functions.GetPlayerData()
-local inRangeCityhall, inRangeDrivingSchool = false, false
+local PlayerData = nil
```

Comment:
Import PlayerData module instead of storing a variable?

Reply:
I like the way you think